src/gui/uithemedialog.ui

```diff
@@ -0,0 +1,298 @@
+          </sizepolicy>
+         </property>
+         <property name="frameShape">
+          <enum>QFrame::NoFrame</enum>
+         </property>
+         <property name="sizeAdjustPolicy">
+          <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
+         </property>
+         <property name="widgetResizable">
+          <bool>true</bool>
+         </property>
+         <widget class="QWidget" name="scrollAreaWidgetContents_2">
+          <property name="geometry">
+           <rect>
+            <x>0</x>
+            <y>0</y>
+            <width>427</width>
+            <height>226</height>
+           </rect>
+          </property>
+          <layout class="QVBoxLayout" name="verticalLayout_4">
+           <item>
+            <layout class="QGridLayout" name="iconsLayout" columnstretch="1,0,0,0,0" columnminimumwidth="0,15,0,15,0">
+             <item row="0" column="0">
+              <widget class="QLabel" name="iconIDLabel">
+               <property name="font">
+                <font>
+                 <bold>true</bold>
+                </font>
+               </property>
+               <property name="text">
+                <string>Icon ID</string>
+               </property>
+              </widget>
+             </item>
+             <item row="0" column="2">
+              <widget class="QLabel" name="lightModeIconLabel">
+               <property name="font">
+                <font>
+                 <bold>true</bold>
+                </font>
+               </property>
+               <property name="text">
+                <string>Light Mode</string>
+               </property>
+              </widget>
+             </item>
+             <item row="0" column="4">
+              <widget class="QLabel" name="darkModeIconLabel">
+               <property name="font">
+                <font>
+                 <bold>true</bold>
+                </font>
+               </property>
+               <property name="text">
+                <string>Dark Mode</string>
+               </property>
+              </widget>
+             </item>
+            </layout>
+           </item>
+           <item>
+            <spacer name="iconsSpacer">
+             <property name="orientation">
+              <enum>Qt::Vertical</enum>
+             </property>
+             <property name="sizeHint" stdset="0">
+              <size>
+               <width>20</width>
+               <height>203</height>
+              </size>
+             </property>
+            </spacer>
+           </item>
+          </layout>
+         </widget>
+        </widget>
+       </item>
+      </layout>
+     </widget>
+    </widget>
+   </item>
+   <item>
+    <widget class="QLabel" name="descriptionLabel">
+     <property name="font">
+      <font>
+       <italic>true</italic>
+      </font>
+     </property>
+     <property name="text">
+      <string>Note: the changes will be fully applied only after the application is restarted.</string>
```

Comment:
I wouldn't say anything because it's GUI related but this may affect translations in the future if i didn't say anything.
This string seems too long (ok with that) but it affects the dialog's minimum width which in turn affects the distance between ID column and the other 2 on the right. In English, the minimum distance is not affected on my machine (barely) but some translations may be excessively long.
I suppose this wouldn't be a problem if the 2 right columns were "left aligned" (don't know if it's possible).
Anyway i would just put a (requires restart) in options and be done with it but it's your choice.

p.s. I didn't test everything but i'm curious: Does changing something without restart affect anything other than new log messages? "the changes will be fully applied" seems like "talking" too much without saying anything.

Nitpick (when resizing vertically):

![Screenshot 2023-03-15 140139](https://user-images.githubusercontent.com/6451685/225303852-8c820398-a0e4-4de2-97c7-ec81a02a7a46.png)

Reply:
> Anyway i would just put a (requires restart) in options and be done with it

Do you mean rename the button to "Customize UI Theme (requires restart)..."?

Reply:
Or just put the label next to it? E.g.: `[Customize UI Theme...] (requires restart)`.

Reply:
Done it in the following way:
![000](https://user-images.githubusercontent.com/5063477/225324682-71785acd-008d-404d-a1ab-ad85c4c89516.png)

P.S. My English still can fail, so please correct the sentence if needed.

Reply:
Funny thing... i just noticed this 
>Changing Interface settings requires application restart

So this is not really needed to be said again huh.

Reply:
> Funny thing... i just noticed this
> 
> > Changing Interface settings requires application restart

🤦

Reply:
Removed!

Reply:
@glassez 
That line also applied to changing the language settings of the interface, with a new line, perhaps not everyone will understand that in order to apply the language settings, you need to restart the application. Are you sure deletion is a good idea? Moreover, it has already existed for a long time and has been translated into many languages.

Reply:
@stalkerok the bottom string seen in this pic was removed https://github.com/qbittorrent/qBittorrent/pull/18655#discussion_r1137083385 exactly because that line exists.